src/main/java/graphql/GraphQL.java

```diff
@@ -284,6 +287,11 @@ public Builder valueUnboxer(ValueUnboxer valueUnboxer) {
             return this;
         }
 
+        public Builder mapFactory(ResponseMapFactory responseMapFactory) {
```

Comment:
@andimarek - should this be @Experimental ??

Reply:
yes it should be

Reply:
and the naming should be `responseMapFactory`

Reply:
done